xcessiv/presets/learnersource.py

```diff
@@ -52,10 +77,64 @@
 base_learner = XGBClassifier(seed=8)
 """
 
-xgboost_regressor_source = \
-    """from xgboost import XGBRegressor
+sklearn_random_forest_regressor_source = \
+    """from sklearn.ensemble import RandomForestRegressor
 
-base_learner = XGBRegressor(seed=8)
+base_learner = RandomForestRegressor(random_state=8)
+"""
+
+sklearn_extra_trees_regressor_source = \
+    """from sklearn.ensemble import ExtraTreesRegressor
+
+base_learner = ExtraTreesRegressor(random_state=8)
+"""
+
+sklearn_bagging_regressor_source = \
+    """from sklearn.ensemble import BaggingRegressor
+
+base_learner = BaggingRegressor(random_state=8)
+"""
+
+sklearn_GP_regressor_source = \
+    """from sklearn.gaussian_process import GaussianProcessRegressor
+
+base_learner = GaussianProcessRegressor(random_state=8)
+"""
+
+sklearn_ridge_regressor_source = \
+    """from sklearn.linear_model import Ridge
+
+base_learner = Ridge(random_state=8)
+"""
+
+sklearn_lasso_regressor_source = \
+    """from sklearn.linear_model import Lasso
+
+base_learner = Lasso(random_state=8)
+"""
+
+sklearn_kernelRidge_regressor_source = \
+    """from sklearn.kernel_ridge import KernelRidge
+
+base_learner = KernelRidge(random_state=8)
+"""
+
+sklearn_knn_regressor_source = \
+    """from sklearn.neighbors import KNeighborsRegressor
+
+base_learner = KNeighborsRegressor(random_state=8)
```

Comment:
I don't think KNN has a random_state parameter. Please also double check the other estimators to see if they have a random_state parameter.